doc/release/RELEASE-NOTES.md

```diff
@@ -41,8 +41,9 @@ Move `countries` to a separate module so that it becomes possible to use both `d
 * [OSDEV-802](https://opensupplyhub.atlassian.net/browse/OSDEV-802) - Distinguish API user and contributor id in the error message that pass to the Rollbar.
 
 ### Release instructions:
-* 0141_delete_contributor_webhooks.py - deletes `ContributorWebhook` model
-
+* Update code.
+* Apply DB migrations up to the latest one.
+* Run the index_facilities_new management command.
```

Comment:
Please, remove` Run the index_facilities_new management command.`. No need to run this command.

Reply:
I see. removed.